### PR TITLE
preserve children during stub

### PR DESCRIPTION
**Pull request by longlho**
@3den

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: lib/helper.js [hunk 1]
```diff
@@ -103,7 +103,9 @@ function stubComponent(tag, children, showDataProps) {
         },
 
         render: function () {
-            return React.createElement(tag, this.getStubProps(), children);
```
Comment (3den) [inline lib/helper.js]: I think we can make simplify a bit 

```js
return React.createElement(tag, this.getStubProps(), this.props.children || children);
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jsx-test",
-  "version": "0.7.1",
+  "version": "0.7.2",
```
Comment (3den) [inline package.json]: move to 0.7.3

File: test/stubComponent.test.js [hunk 3]
```diff
@@ -16,6 +16,13 @@ describe('#stubComponent', function() {
         jsx.assertRender(ComponentStub, {}, '<p>hello world!</p>');
     });
 
+    it('preserves children content the component', function () {
+        var ComponentStub = jsx.stubComponent('p');
+        jsx.assertRender(ComponentStub, {
```
Comment (3den) [inline test/stubComponent.test.js]: :+1:

File: lib/helper.js (hunk 1 above)
Comment (longlho) [inline lib/helper.js]: :+1:

**Comment by longlho** [comment]
@3den done :)

**Comment by 3den** [comment]
published

**Pull request merged**